diff --git a/src/Script/Playerbots.cpp b/src/Script/Playerbots.cpp
index ef39400..69b7e25 100644
--- a/src/Script/Playerbots.cpp
+++ b/src/Script/Playerbots.cpp
@@ -516,6 +516,7 @@ void AddSC_HyjalSummitBotScripts();
 void AddSC_IcecrownBotScripts();
 void AddSC_RubySanctumBotScripts();
 void AddSC_randombot_level_mgr();
+void AddPlayerbotsAiChatScripts();
 
 void AddPlayerbotsScripts()
 {
@@ -536,4 +537,5 @@ void AddPlayerbotsScripts()
     AddSC_IcecrownBotScripts();
     AddSC_RubySanctumBotScripts();
     AddSC_randombot_level_mgr();
+    AddPlayerbotsAiChatScripts();
 }
